### PR TITLE
add ugbio_cloud_utils member; move cloud_sync fromVariantCallling

### PR DESCRIPTION
**Pull request by avigailmo**
source: https://github.com/Ultimagen/VariantCalling/blob/master/ugvc/utils/cloud_sync.py
in use here: https://github.com/Ultimagen/BioinfoResearch/blob/doronst/pp_improvements_nov24/pyproject.toml#L22

### CODE REVIEW

**Review by doron-st — APPROVED** [review]
LGTM

File: src/cloud_utils/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[project]
+name = "ugbio_cloud_utils"
+version = "1.4.3-1"
+requires-python = ">=3.10" #required by some of the consumers (e.g.: ugbir: https://github.com/Ultimagen/BioinfoResearch/blob/master/pyproject.toml#L12)
```
Comment (inbalzelig-ugi) [inline src/cloud_utils/pyproject.toml:4]: change to 3.11

File: src/cloud_utils/README.cloud.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# ugbio_cloud_utils
+
+This module includes cloud (AWS/GS) python utils for bioinformatics pipelines.
```
Comment (inbalzelig-ugi) [inline src/cloud_utils/README.cloud.md:3]: You can add more details about the utils (like cloud_sync script)

File: src/cloud_utils/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
+FROM python:3.10-bullseye AS build
```
Comment (inbalzelig-ugi) [inline src/cloud_utils/Dockerfile:1]: Python 3.11. All our ugbio packages and dockers are 3.11. This is needed for the pre-commit to work

**Review by inbalzelig-ugi — APPROVED** [review]
Please update to python 3.11 to be consistent with the rest of the packages in this repo

**Pull request merged**